src/Task/Base/Watch.php

```diff
@@ -49,15 +49,21 @@ public function __construct($bindTo)
     /**
      * @param string|string[] $paths
      * @param \Closure $callable
+     * @param int|int[] $events
      *
      * @return $this
      */
-    public function monitor($paths, \Closure $callable)
+    public function monitor($paths, \Closure $callable, $events = FilesystemEvent::MODIFY)
     {
         if (!is_array($paths)) {
             $paths = [$paths];
         }
-        $this->monitor[] = [$paths, $callable];
+
+        if (!is_array($events)) {
+            $events = [$events];
+        }
+
+        $this->monitor[] = [$paths, $callable, $events];
```

Comment:
Typecase as `(array) $events` and the `if` block above should not be necessary, I think.

Reply:
You mean it should allow only one event?

Reply:
No; if `$events` is an array, then the statement `(array) $events` is a no-op. If `$events` is a scalar, then `(array) $events` is equivalent to `$events = [$events];`. That's standard but perhaps lesser-known PHP array behavior.

Reply:
Didn't know about this. Thanks.